src/modules/manual_control/ManualControlSelector.cpp

```diff
@@ -67,27 +71,61 @@ bool ManualControlSelector::isInputValid(const manual_control_setpoint_s &input,
 {
 	// Check for timeout
 	const bool sample_from_the_past = now >= input.timestamp_sample;
```

Comment:
is this really necessary? Under what conditions would you have `input.timestamp_sample > now`?

Reply:
Removed in ee325c4b815bfc05161c00b6ec7db5ac3291c393